alembic/script/revision.py

```diff
@@ -964,6 +1034,11 @@ def verify_rev_id(cls, revision):
     def __init__(
         self, revision, down_revision, dependencies=None, branch_labels=None
     ):
```

Comment:
**Koichiro Den** (lkpdn) wrote:

Indeed. Done. PTAL at Patch Set 3.

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2379